docs/source/datastore.rst

```diff
@@ -136,23 +136,23 @@ Get individual key value pair or list all:
 ::
 
     >>> keys = client.keys.get_all()
-    >>> os_keystone_endpoint = client.keys.get_by_name(name='os_keystone_endpoint')
+    >>> os_keystone_endpoint = client.keys.get_by_name(os_keystone_endpoint)
```

Comment:
Good catch.

> st2client/models/core.py
> 281:    def get_by_name(self, name_or_id, **kwargs):

In general I prefer to be explicit and use keyword arguments so another option is to explicitly use `name_or_id` here (although, yeah, most other code doesn't use a keyword argument).